library/core/src/fmt/float.rs

```diff
@@ -228,3 +228,19 @@ macro_rules! floating {
 
 floating! { f32 }
 floating! { f64 }
+
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
It's not the correct version, right?

Reply:
I have just been using the same gate as the traits on `f32` and `f64` for now, since most of the impl macros don't have a way to specify the gate. I think that when this is closer to stabilization I will change the macros and add a separate gate, but for now I don't think it gains much to not just match the other floats.